tpot/tpot.py

```diff
@@ -176,6 +187,10 @@ def __init__(self, population_size=100, generations=100,
         self._pset.addPrimitive(self._select_fwe, [pd.DataFrame, float], pd.DataFrame)
         self._pset.addPrimitive(self._select_percentile, [pd.DataFrame, int], pd.DataFrame)
         self._pset.addPrimitive(self._rfe, [pd.DataFrame, int, float], pd.DataFrame)
+        self._pset.addPrimitive(self._ekf_mask, [pd.DataFrame, pd.DataFrame], pd.DataFrame)
+        self._pset.addPrimitive(self._ekf_select_kbest, [pd.DataFrame, pd.DataFrame, int], 
```

Comment:
I think these two functions should be combined into one. The one function should take a `pd.DataFrame`, an int, and another int as inputs:

* DF = input_df

* first int = which EK source to use

* second int = only used in the "select k best" code when the EK source is not a mask